### DISCUSSION TITLE
Fuzz testing section 12 defi-stablecoin

### DISCUSSION THREAD

**Discussion by EchoSpr** [Cyfrin/foundry-full-course-cu#246] [Q&A] [answered]
When running `forge test --match-test invariant_protocolMustHaveMoreValueThanTotalSupply -vvv` It is not showing that the actual fuzz test is being ran.  The output after running the test is 
 `[⠆] Compiling...
[⠑] Compiling 2 files with 0.8.19
[⠃] Solc 0.8.19 finished in 1.45s
Compiler run successful!

Test result: ok. 0 passed; 0 failed; finished in 8.00ms`

But I noticed in the video instructions after the test has been ran, the command line shows that the actual fuzz test was running.  How can I get the fuzz test to run?

**Comment by PatrickAlphaC** [comment, marked as answer]
Yep! This is an issue I've run into before that's really annoying. I'm not exactly sure when this happens... If you figure it out, can you make an issue on he foundry github? I _think_ it's something to do with your setup not being done correctly.
